activesupport/lib/active_support/cache/strategy/local_cache.rb

```diff
@@ -107,6 +107,11 @@ def cleanup(**options) # :nodoc:
           super
         end
 
+        def delete_matched(key, options = nil)
+          local_cache && local_cache.clear
```

Comment:
I think we should match the implementation of `clear` and `cleanup`, so it's easier to tell that they're doing the same thing:

```suggestion
          return super unless cache = local_cache
          cache.clear
```